scripts/test_nearlib.sh

```diff
@@ -16,19 +16,23 @@ git clone https://github.com/near/create-near-app.git
 # Make sure to use local nearcore for tests
 export NODE_ENV=local
 
+SRC_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+export HOME=$SRC_DIR/../testdir
+
+# Run create-near-app tests
```

Comment:
Why move these to run before near-api-js tests? near-api-js more likely to fail fast.

Reply:
Before Bowen's fix near-api-js was failed and exit, but wanted to test create-near-app on ci. I can move it back now